### PR TITLE
CA-21 Initial Terratests

### PR DESCRIPTION
**Pull request by hbarros-caylent**
This will test the creation of the Elastic Search with the Tamr Sec Group module.
It is almost the same as the example/minimal we have, the difference is in the VPC we need to create to properly run the test.

Additional functions will be added for internal connectivity tests later.

### CODE REVIEW

File: test/create_es_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformCreateES(t *testing.T) {
+	// This creates this function with the possibility of being run in parallel with other functions
+	// that start with this same t.Parallel() call.
+	t.Parallel()
+
+	namePrefix := "terratest-es"
```
Comment (souza-dan) [inline test/create_es_test.go:16]: If tests are running in parallel, will resources conflict?

Should we do something like add a UUID to this value?

File: examples/test_minimal/main.tf [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## If you don't have this ES Role created yet in your AWS Account, this data source will raise an error.
+## To fix the error:
+## - comment this data source,
+## - in the module tamr-es-cluster:
+##     - remove the argument linked_service_role,
+##     - update the argument create_new_service_role = true
+#####
+data "aws_iam_role" "es" {
+  name = "AWSServiceRoleForAmazonElasticsearchService"
+}
```
Comment (souza-dan) [inline examples/test_minimal/main.tf:70]: Very clever!

**Review by souza-dan — COMMENTED** [review]
Overall content here is in great shape!

File: examples/logs/.gitignore [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Local .terraform directories
```
Comment (souza-dan) [inline examples/logs/.gitignore:1]: Do we need a .gitignore specific to this example?

**Review by souza-dan — APPROVED** [review]
LGTM

File: examples/logs/.gitignore (hunk 3 above)
Comment (fvazquez-caylent) [reply examples/logs/.gitignore]: Fixed.

**Pull request merged**